qcengine/programs/entos.py

```diff
@@ -24,6 +25,27 @@ class EntosHarness(ProgramHarness):
     }
     version_cache: Dict[str, str] = {}
 
+    # Energy commands that are currently supported
+    _energy_commands: Set[str] = {"dft",
+                                  # "xtb"
+                                  }
+
+    # List of DFT functionals
+    _dft_functionals: Set[str] = {
+        "SLATER", "DIRAC", "SLATERD3", "DIRACD3", "VWN5", "VWN", "VWN1", "SVWN", "LDA", "BLYP", "BPW91",
+        "BLYPD3", "B88", "PBEX", "PBERX", "PBEC", "LYP", "PW91", "P86", "PBE", "PBER", "PBED3", "PBERD3",
+        "B3LYP3", "B3LYP", "B3LYP5", "PBE0", "PBE1PBE", "B3LYP3D3", "B3LYPD3", "B3LYP5D3", "PBE0D3",
+        "PBE1PBED3", "CAMB3LYP", "WB97X", "CAMB3LYPD3", "WB97XD3"
+    }
```

Comment:
Ouch, any way to avoid this in the future you think?

Reply:
Hmm potentially. Entos provides their documentation in an xml format which does contain this list of functionals under the `xc` option tag. It could be parsed from there as long as the documentation file is always accessible. I'm not sure if this will always be the case depending on how the code is distributed.